Correctly resolve WebPack loaders within npm linked modules

### OS?
OSX Sierra

### Versions.
angular-cli: 1.0.0-beta.18
node: 6.9.1
os: darwin x64

### Repro steps.
Install NPM linked package that uses a WebPack loader. Loader cannot be resolved.

### The log given by the failure.
`Module not found: Error: Can't resolve 'style' in '...'`


### Mention any other details that might be useful.
Merge #2291 was incomplete, in that it does not fully address the issue raised in #1875. It fixes _module_ resolution, but not _loader_ resolution. The following needs to be added to the object returned from `getWebpackCommonConfig()` in `webpack-build-common.ts` in order to match current module resolution strategy:
```
resolveLoader: {
  modules: [path.resolve(projectRoot, 'node_modules')]
}
```


Also, I would submit a pull request, but the the corporate overhead of getting a CLA in place are prohibitive.

--- a/packages/angular-cli/models/webpack-build-common.ts
+++ b/packages/angular-cli/models/webpack-build-common.ts
@@ -58,9 +58,12 @@ export function getWebpackCommonConfig(
     devtool: sourcemap ? 'source-map' : false,
     resolve: {
       extensions: ['.ts', '.js'],
-      modules: [nodeModules]
+      modules: [nodeModules],
     },
-    context: path.resolve(__dirname, './'),
+    resolveLoader: {
+      modules: [path.resolve(projectRoot, 'node_modules')]
+    },
+    context: projectRoot,
     entry: entry,
     output: {
       path: path.resolve(projectRoot, appConfig.outDir),

--- a/packages/angular-cli/models/webpack-build-test.js
+++ b/packages/angular-cli/models/webpack-build-test.js
@@ -56,7 +56,7 @@ const getWebpackTestConfig = function (projectRoot, environment, appConfig, test
 
   return {
     devtool: testConfig.sourcemap ? 'inline-source-map' : 'eval',
-    context: path.resolve(__dirname, './'),
+    context: projectRoot,
     resolve: {
       extensions: ['.ts', '.js'],
       plugins: [

